dask/tests/test_core.py

```diff
@@ -140,14 +140,6 @@ def test_get_dependencies_task_none():
 
 
 def test_get_deps():
-    """
-    >>> dsk = {'a': 1, 'b': (inc, 'a'), 'c': (inc, 'b')}
-    >>> dependencies, dependents = get_deps(dsk)
-    >>> dependencies
-    {'a': set(), 'b': {'a'}, 'c': {'b'}}
-    >>> dependents  # doctest: +SKIP
-    {'a': {'b'}, 'b': {'c'}, 'c': set()}
-    """
```

Comment:
The function being tested already has doctests.

https://github.com/dask/dask/blob/d6bbbb08c92652eae2820e93edc2f3fe502391d3/dask/core.py#L261-L270